.env.example

```diff
@@ -35,3 +35,10 @@ MI_ABSENCE_MANAGEMENT="https://<server>/<resource-path>"
 MI_TRAVEL_AND_EXPENSE="https://<server>/<resource-path>"
 MI_GSD="https://<server>/<resource-path>"
 
+# Agentforce config
+# Set the following environment variables to connect to the Agentforce API.
+# See https://developer.salesforce.com/docs/einstein/genai/guide/agent-api-get-started.html for more information.
+AGENTFORCE_MY_DOMAIN_URL="https://<DOMAIN_NAME>.my.salesforce.com"
+AGENTFORCE_AGENT_ID="<AGENT_ID>"
+AGENTFORCE_CLIENT_ID="<CLIENT_ID>"
+AGENTFORCE_CLIENT_SECRET="<CLIENT_SECRET>"
```

Comment:
Added the list of env variables to the .env.example for a simpler way to set them up